tests/conftest.py

```diff
@@ -70,31 +70,31 @@ def fridge_ds(samples_source, fridge_class_map) -> Tuple[Dataset, Dataset]:
         class_map=fridge_class_map,
     )
 
-    data_splitter = RandomSplitter([0.8, 0.2])
+    data_splitter = RandomSplitter((0.5, 0.5))
     train_records, valid_records = parser.parse(data_splitter)
 
     tfms_ = tfms.A.Adapter([A.Resize(IMG_SIZE, IMG_SIZE), A.Normalize()])
 
-    train_ds = Dataset(train_records[:2], tfms_)
```

Comment:
I'm a bit hesitant of this change (removing `[:2]`) because I remember I added it so the models training check wouldn't take long, specially on the github CI that is CPU only.

But but, it seems it took 7m this time, so it should be fine

Reply:
currently the fridge test sample only has 6 items so I thought it would be ok. I removed it to be able to test batch_size=3